versions.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  required_version = "~> 0.12.13"
```

Comment:
since we're requiring this version or higher patch do we want to set the version for the CI to 0.12.13 instead? or is there a reason to have the difference?

Reply:
Excellent catch. I think locally my version is outdated compared to latest (which is what I snagged for CI). I can bump that down. The particular patch really doesn't matter afaik for this PR, this was just auto generated.